cfgov/data_research/management/commands/conference_export.py

```diff
@@ -0,0 +1,185 @@
+from __future__ import print_function
+
+import csv
+import json
+import logging
+
+from cStringIO import StringIO
+from django.core.mail import EmailMessage
+from django.core.management.base import BaseCommand
+from django.utils.encoding import force_bytes
+from wagtail.wagtailcore.models import Page
+
+from data_research.models import ConferenceRegistration
+from v1.util.migrations import get_stream_data
+
+
+logger = logging.getLogger(__name__)
+
+
+def get_registration_form_from_page(page_id):
+    page = Page.objects.get(pk=page_id).specific
+    content = get_stream_data(page, 'content')
+
+    for block in content:
+        if 'conference_registration_form' == block['type']:
+            return block
+
+    raise RuntimeError('no registration form found on {}'.format(page))
+
+
+class ConferenceExporter(object):
+    FIELDS = (
+        ('name', 'Name'),
+        ('organization', 'Organization'),
+        ('email', 'Email Address'),
+        ('sessions', 'Selected Sessions'),
+        ('foodinfo', 'Food Restrictions'),
+        ('accommodations', 'Accommodations Needed'),
+        ('code', 'GovDelivery Code'),
+    )
+
+    EMAIL_SUBJECTS = (
+        '[2016 CFPB Research Conference] Attendance Update',
+        '[2016 CFPB Research Conference] Attendance Full',
+    )
+
+    EMAIL_BODIES = (
+        (
+            'The 2016 Research Conference is at {0.count} of '
+            '{0.conference_capacity} for signups.'
+        ),
+        (
+            'The 2016 Research Conference has reached capacity with {0.count} '
+            'submissions. Signups are now closed.\n\nThe capacity for this '
+            'event was set to {0.conference_capacity} at the time of this '
+            'email.'
+        ),
+    )
+
+    EMAIL_FOOTER = (
+        '\n\nSee attached for attendee details.\n\n'
+        'This is an automated message.'
+    )
+
+    ATTACHMENT_FILENAME = 'conf_reg.csv'
```

Comment:
Can you make the filename something friendlier? Like `conference_registrations.csv`

Also thinking a nice enhancement would be to timestamp it. *adds to backlog*